hadoop-ozone/common/pom.xml

```diff
@@ -30,6 +30,21 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
 
   <dependencies>
 
+    <dependency>
+      <groupId>io.grpc</groupId>
```

Comment:
Most of Ozone modules use shaded grpc/netty from ratis third party library work . Is there a reason to bring the unshaded grpcy/netty dependency?

Reply:
Thanks for reviewing this @xiaoyuyao and for your comments.  On the using the ratis grpc/netty libraries, initially I wanted to use them, however they were not compatible with the OzoneManagerProtocol we are realizing and using.  Works with the unshaded grpc/netty versions ( tested with 4.1.52.Final and 4.1.63.Final ) together with protoc compiler (version 2.5.0 and NOT 3.12.0) without issue.  

An example of the incompatibility includes the OzoneManagerProtocol interfacing with the HddsProtos with the thirdparty libraries.

Reply:
@neils-dev Can you please provide more specific examples (e.g. compile errors or runtime exceptions)?  Do you have another branch or commit where the shaded libraries are used?  These would help us understand the issues.  Thanks.

Reply:
@adoroszlai, as suggested see attached for sample errors when using shaded libs.
[protoc_generated_errors_shaded.txt](https://github.com/apache/ozone/files/6803515/protoc_generated_errors_shaded.txt)

Reply:
Thanks, but as mentioned, it would be more useful with the code that resulted in these errors.

Reply:
I'll provide a link to a branch you can view and compile from...

Reply:
@adoroszlai : code resulted in error; commit with shaded libraries used found in local branch https://github.com/neils-dev/ozone/tree/HDDS-5210_thirdPartyProtoc.  Ozone build should give you a sample of errors. Thanks.